package.json

```diff
@@ -28,6 +28,7 @@
         "typescript": "~4.5.4"
     },
     "dependencies": {
+        "@aws-quickstart/eks-blueprints": "file:/Users/pevetoej/environment/cdk-eks-blueprints/aws-quickstart-eks-blueprints-1.0.5.tgz",
```

Comment:
drop this change, it was needed for the patterns repo and we made a mistake to run npm i here.